fpdf/template.py

```diff
@@ -190,6 +191,7 @@ def _varsep_float(s, default="0"):
             ("priority", int, False),
             ("multiline", self._parse_multiline, False),
             ("rotate", _varsep_float, False),
+            ("keep_aspect_ratio", int, False),
```

Comment:
```suggestion
            ("keep_aspect_ratio", bool, False),
```

Reply:
Same as above, the converter is `int` here in order to maintain similarity to the other boolean flags. Would need to test if this behaviour works (though I cannot see why it shouldn't).

Reply:
This may have been an attempt at backwards compatibility at the time.
I don't think we still support any Python versions without bools though, so it probably makes sense to change all relevant items to use `bool` now.

Reply:
Agreed!

Do you have enough information in our answers to adress those points @Clock-Speed? 🙂